src/Components/ListingCard.js

```diff
@@ -1,8 +1,11 @@
 import axios from "axios";
 import React, { useEffect, useState } from "react"
 import { Card } from 'react-bootstrap';
+import Modal from 'react-bootstrap/Modal'
+import Button from 'react-bootstrap/Button'
 
 export const ListingCard = (props) => {
+    const [modalShow, setModalShow] = React.useState(false);
```

Comment:
since useState is imported already, don't need to write React.useState, can just do useState